### PR TITLE
 vkd3d: Enable binary import and image view handle extensions

### PR DESCRIPTION
**Pull request by rgc183**
This also updates subprojects/Vulkan-Headers to 1.2.180

This is needed for VK_NVX_binary_import and VK_NVX_image_view_handle.

Signed-off-by: Roshan Chaudhari <rochaudhari@nvidia.com>

Reviewed-by: Liam Middlebrook <lmiddlebrook@nvidia.com>

### CODE REVIEW

File: include/vkd3d_win32.h [hunk 1]
```diff
@@ -42,9 +42,20 @@
 #define WIDL_C_INLINE_WRAPPERS
 #include <vkd3d_windows.h>
 
+/* Vulkan headers include static const declarations. Enable static keyword for
```
Comment (liam-middlebrook) [inline include/vkd3d_win32.h:45]: Could you explain more why this is necessary? It's not really clear to me. Why is is just ifdef'd for MINGW32? What is specific about MINGW32 that causes issues here?

Comment (rgc183) [reply include/vkd3d_win32.h:45]: line 32, makes hack for MINGW32 and causes `static` to meaningless. So we undo this hack for vulkan headers. If we do not, header files will have declaration of variables without static keyword and then linking with multiple libraries will throw "multiple definition"

Comment (liam-middlebrook) [reply include/vkd3d_win32.h:45]: Could we move the inclusion of the Vulkan headers before this hack? Or does that cause issues with the WIN32 platform definition?

Comment (rgc183) [reply include/vkd3d_win32.h:45]: vulkan header is calling windows.h which eventually calls to winbase.h in mingw, and it fails to find declaration for `IUnknown_**` .

Comment (HansKristian-Work) [reply include/vkd3d_win32.h:45]: Right, we get silly stuff like this if we don't do this hack:
```
/usr/lib/gcc/x86_64-w64-mingw32/11.1.0/../../../../x86_64-w64-mingw32/bin/ld: libs/vkd3d/libvkd3d-proton.a(swapchain.c.obj):swapchain.c:(.rdata+0x29a8): multiple definition of `VK_PIPELINE_STAGE_2_TOP_OF_PIPE_BIT_KHR'; libs/d3d12/d3d12.dll.p/main.c.obj:main.c:(.rdata+0x8d0): first defined here
/usr/lib/gcc/x86_64-w64-mingw32/11.1.0/../../../../x86_64-w64-mingw32/bin/ld: libs/vkd3d/libvkd3d-proton.a(swapchain.c.obj):swapchain.c:(.rdata+0x29b0): multiple definition of `VK_PIPELINE_STAGE_2_NONE_KHR'; libs/d3d12/d3d12.dll.p/main.c.obj:main.c:(.rdata+0x8d8): first defined here
```
We already do stupid hacks for MinGW here, so I guess I'll let this slide.

**Review by HansKristian-Work — APPROVED** [review]
LGTM

**Pull request merged**